packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -918,10 +918,23 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
         this._logger = ChildLogger.create(this.logger, "ContainerRuntime");
 
+        /**
+         * Function that return the current server timestamp. This is used by the garbage collector to set the
+         * time when a node becomes unreferenced.
+         * For now, we use the timestamp of the last op for gcTimestamp. However, there can be cases where
+         * we don't have an op (on demand summaries for instance). In those cases, we will use the timestamp
+         * of this client's connection - https://github.com/microsoft/FluidFramework/issues/7152.
```

Comment:
This issue is closed. Is there another issue tracking this work?

Reply:
Good catch. Created a new issue for this - https://github.com/microsoft/FluidFramework/issues/8375